src/schemas.ts

```diff
@@ -124,7 +124,7 @@ export const MentorApplicationSchema = z.object({
 });
 
 export const MenteeCheckInSchema = z.object({
-  title: z.string().min(1, 'Title is required'),
+  title: z.string().min(1, 'Month is required'),
```

Comment:
```suggestion
  month: z.string().min(1, 'Month is required'),
```
Change the field as well